app/src/components/Layout/Box.js

```diff
@@ -0,0 +1,109 @@
+import { useState, Fragment, useRef } from "react";
+
+import Form from "../Forms/Form";
+import Button from "../UI/Button";
+import ErrorModal from "../UI/ErrorModal";
+import { search } from "../../CreateExcelFile/GoogleSearch";
+import { createExcel } from "../../CreateExcelFile/ExcelUtils";
```

Comment:
[optional]
this is fine, but what i personally like to do is:
import * as googleService from "../../CreateExcelFile/GoogleSearch";

after that, whatever you gotta use, u can just go...
googleService.search()
googleService.ping()
etc etc.

point being, makes for cleaner code when interacting with different modules

exact same thing goes for the excelutils...
import * as excelUtils from "../../CreateExcelFile/ExcelUtils";
then ...excelUtils.createExcel(...)

Reply:
OK yea I might go with this approach...